.github/workflows/ci.yml

```diff
@@ -2,9 +2,9 @@ name: CI
 
 on:
   push:
-    branches: [master]
+    branches: [nikita/permissionless-pool]
   pull_request:
-    branches: [master]
+    branches: [nikita/permissionless-pool]
```

Comment:
keeping this and committing it will break other branches merging against master. i would imagine there is a solution to run against all branches in the `saddle-frontend` repo if that's what we want. what do you think @hammeiam ?

Reply:
@ug02fast  this solution works. 
https://stackoverflow.com/questions/64635032/github-actions-run-on-push-to-all-branche

Reply:
great, unless @hammeiam has any objections it might be a good idea to change